requirements/base.txt

```diff
@@ -1,2 +1,3 @@
-neo4j>=4.4.2, <5.0
-pandas>=1.0, <2.0
+multimethod >= 1.0, < 2.0
```

Comment:
kinda wonder if its not safer and sufficient to require a specific version. or would that risk versions being deprecated and then fail to be fetched ?

Reply:
Because of how people install things with pip, the convention is to specify an as generous interval as possible here. Several packages a user have installed in their Python environment may have the same `multimethod` dependency. If they all had a different version of `multimethod` fixed, only one package could be installed unless they were the same. One solution is of course to use something like virtual environments, but not all people do. And having one VE for each package installation is rather steep I think.

Since we're just within one major version here, it should at least in theory not be an issue for us. If we were to support multiple major version for some dependency, we'd have to make to to add every such major version with tox.